device-types/Cisco/ASR-9902.yaml

```diff
@@ -0,0 +1,143 @@
+---
+manufacturer: Cisco
+model: ASR-9902
+slug: asr-9902
+part_number: ASR-9902
+u_height: 2
+is_full_depth: true
+subdevice_role: parent
+comments: '[Cisco ASR 9000 Series Aggregation Services Routers Data Sheet](https://www.cisco.com/c/en/us/products/collateral/routers/asr-9000-series-aggregation-services-routers/data_sheet_c78-501767.html)'
+module-bays:
+- name: PM0
+  label: ''
+  position: '0'
+  description: Power module
+- name: PM1
+  label: ''
+  position: '1'
+  description: Power module
+- name: RP0
+  label: ''
+  position: '0'
+  description: Route Processor
+- name: RP1
+  label: ''
+  position: '1'
+  description: Route Processor
+- name: FAN0
+  label: ''
+  position: '0'
+  description: Fan tray
+- name: FAN1
+  label: ''
+  position: '1'
+  description: Fan tray
+- name: FAN2
+  label: ''
+  position: '2'
+  description: Fan tray
+
+---
```

Comment:
Please correct this file, it looks like you have two seperate files in one. Only one device should be defined per file